packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -720,7 +720,7 @@ export async function proxyRequestToAxios(
 					error: responseData,
 					response: pick(response, ['headers', 'status', 'statusText']),
 				});
-			} else if (error instanceof Error && error.message.includes('SSL routines')) {
+			} else if ('rejectUnauthorized' in configObject && error.code?.includes('CERT')) {
```

Comment:
Yes, rejectUnauthorized limits the error to HTTP Request node and a couple of other nodes.

Is there why we need to change `error.message.includes('SSL routines')`? Does the `error.code?.includes('CERT')` matches more SSL issue errors?

Reply:
I tested this using a few urls from https://badssl.com/ and looks like axios catches some SSL issues and rewrites `error.message`. `error.code` was a lot more reliable as a check.